server/src/main/java/io/druid/guice/http/HttpClientModule.java

```diff
@@ -135,14 +137,24 @@ public HttpClient get()
     {
       final DruidHttpClientConfig config = getConfigProvider().get().get();
 
+      if (config.getUnusedConnectionTimeout() != null &&
+          config.getUnusedConnectionTimeout().isLongerThan(config.getReadTimeout())) {
+        LOG.warn(
```

Comment:
another idea would be to have `unusedConnectionTimeout` feature disabled by default (by having null value instead of 4 minutes) and then we can change this warn into an exception. I think that is preferable as nobody would really pay attention to log.

Reply:
Sounds good, 👍